apps/frontend-pwa/src/components/learningElements/SingleQuestion.tsx

```diff
@@ -108,12 +108,6 @@ function SingleQuestion({
           <div className="flex flex-row items-center justify-between mb-4 border-b">
             <H3 className={{ root: 'mb-0' }}>{questionData.name}</H3>
             <div className="flex flex-row items-center gap-3">
-              <div className="text-sm md:text-base text-slate-500">
```

Comment:
removed this as it is quite buggy in stacked learning elements

Reply:
It is not really buggy, just hasn't been fixed since the switch from single questions to stacks :) If we remove this, we should probably also remove the otherwise unused props (currrentStep and totalSteps) as well as the corresponding translation string-.